### DISCUSSION TITLE
Any way we can obtain the keymap layout in a pdf format?

### DISCUSSION THREAD

**Discussion by mdkul22** [nickcoutsos/keymap-editor#258] [Q&A] [answered]
First of all, the keymap editor is great, it's really made it easy to configure my corne setup for zmk. I was wondering if it would be possible to be able to save a cheatsheet of sort using the keymap editor regarding all the layers in one page.

**Comment by nickcoutsos** [comment, marked as answer]
A lot of people have asked for this and after a few false starts I really haven't had time/ambition to push forward on that one. Right now your best bet is to check out Keymap Drawer -- there's a GithHub workflow you can add to your repo so that it automatically generates and saves a diagram whenever you commit changes. I'm not sure if PDF is a supported option but if not you may be able to find a way to convert a generated PNG/SVG.

**Reply by mdkul22** [reply]
Thanks a lot!
